docs/project/2024/roman.gulida/index.md

```diff
@@ -35,30 +35,47 @@ I've been interested in cryptocurrencies for a long time and use price tracking
     A web server running on the Pico WH serves a simple web page. This web page allows remote monitoring of prices and setting of price alerts.
 
 * Connections and Flow 
-  + The Pico W fetches data from the CoinMarketCap API at regular intervals, typically using HTTP requests. This data is processed to extract the relevant price and daily   percentage change. 
+  + The Pico W fetches data from the CoinMarketCap API at regular intervals, typically using HTTP requests. This data is processed to extract the relevant price and daily percentage change. 
   + The screen is updated with the latest data, displaying the current cryptocurrency price and daily change. 
   + The toggle switch allows users to select which cryptocurrency to track, changing the request to the CoinMarketCap API accordingly. 
   + The buzzer is triggered when price alerts are activated. The alert thresholds are set via the web page, which communicates with the Pico WH through the web server. 
   + The web server running on the Pico WH allows users to access the tracker remotely through a web page, providing a convenient interface for monitoring prices and configuring alerts.
 
 ## Log
 
-
 <!-- write every week your progress here -->
 
 ### Week 6 - 12 May
-
+I received the hardware package, verified its contents, and started assembling the components. 
 ### Week 7 - 19 May
-
+I completed the hardware assembly. I finished the KiCad schematic and began developing the software. This involved work with LCD and creating structs CoinData and MarketData.
 ### Week 20 - 26 May
 
 ## Hardware
 
 In my project I used a 1.8-inch SPI LCD with the ST7735 controller (128 x 160 px) to display cryptocurrency prices and daily percentage changes. The central microcontroller is the Raspberry Pi Pico WH. I used a breadboard for prototyping, a 3V active buzzer for auditory price alerts, toggle switch for switching between currencies and jumper wires to connect everything.
 
++ Display(data would be displayed in format "st7735 Pin -> Raspberry Pi Pin"):
+  * VCC -> 3V3(OUT), pin 36
+  * SCL -> GPIO 10, pin 14
+  * SDA -> GPIO 11, pin 15
+  * RES -> GPIO 12, pin 16
+  * DC -> GPIO 8, pin 11
+  * CS -> GPIO 9, pin 12
+  * GND -> pin 38
+  * BL -> None
+
++ Buzzer. The buzzer is connected to the GPIO 0(pin 1) and ground(pin3).
++ Push button. Pushbutton switch is connected to the GPIO 3(pin 5) and ground(pin 8).
+ 
+
+Images of the physical hardware and connections:
+![hardware1](hardware1.jpg)<br> 
+![hardware2](hardware2.jpg)
```

Comment:
The <br> tag on the line 73 is causing your code to fail test deployment checks.
If you want to break a line in text, rather than using the <br> tag, I would suggest using '&nbsp', in the following way:
//For leaving one space
//-[hardware1](hardware1.jpg)
//&nbsp
//
//-[hardware2](hardware2.jpg)

or
//For leaving two spaces
//-[hardware1](hardware1.jpg)
//&nbsp
//
//&nbsp
//-[hardware2](hardware2.jpg)
And so on..